ee/vellum_ee/workflows/display/tests/workflow_serialization/generic_nodes/test_adornments_serialization.py

```diff
@@ -271,45 +271,45 @@ class StackedWorkflow(BaseWorkflow):
             "ports": [{"id": "408cd5fb-3a3e-4eb2-9889-61111bd6a129", "name": "default", "type": "DEFAULT"}],
             "adornments": [
                 {
-                    "id": "3344083c-a32c-4a32-920b-0fb5093448fa",
+                    "id": "5be7d260-74f7-4734-b31b-a46a94539586",
```

Comment:
revert the `id` changes in this file. I know that means this test will fail. I want to then pull it down to see how I can get it to pass without updating these ids

Reply:
reviewing the id changes, they are undesirable here